pgx-pg-sys/src/submodules/setjmp.rs

```diff
@@ -0,0 +1,37 @@
+#[inline(always)]
+pub unsafe fn postgres_function_guard<T, F: FnOnce() -> T>(f: F) -> T {
```

Comment:
I guess `pg_guard` works but also doesn't hint to me what it's guarding?

Reply:
Maybe we make this `pub(crate)` for now.  Since we wholesale apply this anyways, users won't need to use it.

Down the road if we make `#[pg_guard]` optional, then we can expose this to the world.

Reply:
`pub(crate) unsafe fn pg_guard_ffi_boundary(...)` is the new spelling.